dashboard/test/controllers/api/v1/sections_controller_test.rb

```diff
@@ -84,6 +84,18 @@ class Api::V1::SectionsControllerTest < ActionController::TestCase
     assert_equal expected, returned_json
   end
 
+  test 'returns all sections instructed by teacher' do
+    cotaught_section = create(:section, user: create(:teacher), login_type: 'word')
+    create(:section_instructor, instructor: @teacher, section: cotaught_section, status: :active)
```

Comment:
Should we also test that a section where the user is the owner but section_instructor does not exist is still returned?

Reply:
Done. This test will need to be deleted once the post-backfill code cleanup is done, but for now it's important to verify that these old sections still remain in the list.